cmake/modules/GetGitRevisionDescription.cmake

```diff
@@ -86,44 +86,130 @@ function(get_git_head_revision _refspecvar _hashvar)
       PARENT_SCOPE)
 endfunction()
 
-function(git_describe _var)
+function(git_get_latest_tag _var)
```

Comment:
Added 2 new functions: 1 to fetch latest tag in repository, the other to fetch commit delta between latest tag and HEAD.